data/resources/ui/login.ui

```diff
@@ -218,25 +218,12 @@
                             <property name="title" translatable="yes">Scan Code</property>
                             <property name="description" translatable="yes">Scan this code with another Telegram app logged into your account.</property>
                             <child>
-                              <object class="GtkFrame">
-                                <property name="width-request">200</property>
-                                <property name="height-request">200</property>
+                              <object class="GtkImage" id="qr_code_image">
                                 <property name="halign">center</property>
-                                <property name="valign">center</property>
-                                <property name="child">
-                                  <object class="AdwBin" id="qr_code_bin">
```

Comment:
Mh, thinking about that, maybe the intent was to not hide the entire card, but only hiding the image?

Reply:
Actually, the fact that the qrcode hides when pressing the 'previous' button is not really great to see, I'll change that.

Reply:
Agreed, hiding it was not the best idea.
Could we then at least apply a simple brightness filter? That makes it look more consistent when all other elements are insensitive.

for example applying this
```css
.insensitive-image {
  filter: brightness(30%);
}
```
has quite good results with not too much effort.
![Bildschirmfoto von 2022-01-07 07-51-17](https://user-images.githubusercontent.com/3630213/148504160-853c8b1d-3c88-4e43-a2f7-3ff0f31fb8c2.png)

Reply:
I just copied what libadwaita does (and also the difference in the hc style).